hist/hist/inc/TGraphErrors.h

```diff
@@ -55,6 +55,7 @@ class TGraphErrors : public TGraph {
    TGraphErrors(const TH1 *h);
    TGraphErrors(const char *filename, const char *format="%lg %lg %lg %lg", Option_t *option="");
    ~TGraphErrors() override;
+   virtual void    AddPointError(Double_t x, Double_t y, Double_t ex = 0., Double_t ey = 0.); ///< Append a new point with errorbars (default to zero) to the graph.
```

Comment:
May be it should be called `AddPointErrors` (with an `s`) ...

Reply:
Or AddPointWithErrors ?

Reply:
Indeed this extra `s` is not a good idea. because we have SetPointError (without `s`):
https://root.cern/doc/v630/classTGraph2DErrors.html#a8001e8d7927ed38801cfc72ef24f8463

My bad ... sorry